manifests/l/Lux/App/1.2.3.4/Lux.App.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Lux.App
+PackageVersion: 1.2.3.4

```

```suggestion
PackageVersion: 1.39.0
```
Please update file path to match this change.